deeplake/core/vectorstore/deep_memory.py

```diff
@@ -398,7 +398,7 @@ def evaluate(
 
         if not INDRA_INSTALLED:
             raise ImportError(
-                "indra is not installed. Please install indra to use this functionality with: pip install `deeplake[enterprise]`"
+                "The C++ library is not installed."
```

Comment:
We should add. 'The library should be installed using `pip install deeplake`, but if you want to install it separately, you may run `pip install libdeeplake`.'